### PR TITLE
[Notifier] [FakeSms] Add the bridge

### PR DESCRIPTION
**Pull request by JamesHemery**
Docs for symfony/symfony#39949

### CODE REVIEW

File: notifier.rst
Comment (BackEndTea) [inline notifier.rst]: Maybe it would be better to have a `since` column in the table, instead of adding a new `versionadded` every new minor?

Though thats something the maintainers should decide

Comment (JamesHemery) [reply notifier.rst]: I also think it's a good idea, @OskarStark ?

Comment (OskarStark) [reply notifier.rst]: @javiereguiluz I tend to agree and like the idea but the problem is the table width, right now it is already to width

File: notifier.rst
Comment (OskarStark) [inline notifier.rst]: This looks wrong 😄

File: notifier.rst
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The Iqsms, GatewayApi, Octopush, AllMySms, Clickatell, SpotHit and FakeSms integrations
```

**Review by OskarStark — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
It took time, but here we go, this is in now. Thank you very much James.

**Pull request merged**
